packages/contracts/src/dollar/core/DollarMintExcess.sol

```diff
@@ -37,25 +37,25 @@ contract DollarMintExcess is IDollarMintExcess {
         if (excessDollars > _minAmountToDistribute) {
             address treasuryAddress = manager.treasuryAddress();
 
-            // curve UbiquityDollar-3CRV liquidity pool
+            // curve uAD-3CRV liquidity pool
```

Comment:
```suggestion
            // curve UbiquityDollar-3CRV liquidity pool
```

Looks like the old changes were correct from the renames. I included "Ubiquity" here to reduce ambiguity because its referring to other protocol tokens in the same sentence.

Reply:
done